src/pages/Home/index.tsx

```diff
@@ -1,14 +1,40 @@
 import { Container } from 'react-bootstrap';
-
+import { useEffect, useState } from 'react';
+import { OrderProps, BobaProps } from 'types/common/main';
+import { bobaList } from 'assets/sampleBobaAPI';
 import BobaContainer from 'components/BobaContainer';
-import { OrderProps } from 'types/common/main';
+import SearchBar from 'components/SearchBar';
 
 const Homepage = (props: OrderProps): JSX.Element => {
   const { order, setOrder } = props;
```

Comment:
```const Homepage = ({ order, setOrder }: OrderProps): JSX.Element => {
  ...
}
```
Refactor suggestion

Reply:
I am using Props on almost every props that i pass on react, should i refactor those too then ?

Reply:
Yep